server/ruby/config_helper.rb

```diff
@@ -0,0 +1,264 @@
+# This utility is to help ensure the sample is configured correctly when first
+# getting started.
+#
+# Refer to `server.rb` for integration details.
+require 'stripe'
+
+class ConfigHelper
+  REQUIRED_VARS = [
+    'BASIC_PRICE_ID',
+    'DOMAIN',
+    'PRO_PRICE_ID',
+    'STATIC_DIR',
+    'STRIPE_PUBLISHABLE_KEY',
+    'STRIPE_SECRET_KEY',
+    'STRIPE_WEBHOOK_SECRET',
+  ]
+
+  def self.check_env!(*args)
+    helper = self.new(*args)
+
+    # Confirms the required environment variables have
+    # been configured.
+    valid = helper.dotenv_exists?
+
+    # Confirms API keys are set and have the expected
+    # prefixes.
+    valid &= helper.valid_api_keys?
+
+    exit if !valid
+
+    # Once we've done basic key validation, we can set the API
+    # key and make deeper assumptions.
+    Stripe.api_key = ENV['STRIPE_SECRET_KEY']
+
+    exit if !helper.valid_paths?
+    exit if !helper.valid_domain?
+
+    exit if !helper.valid_prices?
+  end
+
+  def initialize(*args)
+    @vars = Dotenv.parse(*args)
+  end
+
+  def write_dotenv!
+    File.open('.env', 'w') do |f|
+      REQUIRED_VARS.each do |v|
+        f.puts "#{v}=#{@vars[v]}"
+      end
+    end
+  end
+
+  def valid_prices?
+    _valid_price?('BASIC_PRICE_ID', 1200) &&
```

Comment:
Why would you want to have these values hard-coded here? Does the price matter?

Reply:
It looks like the `unit_amount` param isn't used. Was the idea here to provide the default value to _generate_ the price with when it doesn't already exist? (Rather than validating the provided price ID, which I what I was thinking initially).

Reply:
So I hard coded the amount here because it's hard coded on the front end in index.html. 

I could definitely see a this expanded to be some steps where we ask: "what currency do you want to charge in? (i.e. USD)" and "how much do you want to charge? (specified in cents like 1200)" and capturing those as part of this helper to create the price. 

We could also fetch a list of existing prices and say: Do you want to use one of these existing prices or create a new one?